util/configuration.py

```diff
@@ -279,3 +281,33 @@ def update(cls):
                         line = 'rcon.password={}\n'.format(cls.password)
                     
                     fileOut.write(line)
+
+class Temp:
+    __data = File.data.get("Temp", {})
+    current = __data.get('current', None)
+    date = __data.get('date', None)
+    elapsed = __data.get('elapsed', None)
+    maximum = __data.get('maximum', None)
+    minutes = __data.get('minutes', None)
+    overheating = __data.get('overheating', False)
+
+    @classmethod
+    def build(cls):
+        print('Because I\'m running on a Raspberry Pi, core temperature can be worrysome so I monitor my core temperature for you.')
+        cls.maximum = ci.int_input('So, how hot should I let myself get?', default=70)
+        cls.minutes = ci.int_input('And how long should I be at this temp before I shut everything down?', default=3)
```

Comment:
Do you think we should be more explicit in this message? Like clarifying we mean minutes or whatever measurement we are using?